lepotato/overlays/etc/kernel/postinst.d/uboot

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+echo "kernel u-boot: install kernel link and dtb" >&2
+ln -sf $(basename $2) /boot/Image > /dev/null 2>&1
+cp -a /usr/lib/linux-image-$1 /boot/dtb > /dev/null 2>&1
+exit 0
```

Comment:
same you can tell u-boot to use a directory of fdt files if the u-boot enable has been done correctly